script/tasks/solidity/transferOwnershipOfPeripheryContractsForImmutable.s.sol

```diff
@@ -42,17 +42,14 @@ contract DeployScript is Script {
             "json"
         );
         networkLogJSON = vm.readFile(path);
-        console.log("A");
```

Comment:
nit: `console` should be removed from import as well.

Reply:
removed